python-ufonormalizer/PKGBUILD

```diff
@@ -4,14 +4,15 @@
 pkgname=python-ufonormalizer
 _pyname=${pkgname#python-}
 pkgver=0.6.1
-pkgrel=3
+pkgrel=4
 pkgdesc='A tool that will normalize XML and other data inside of a UFO'
 arch=(any)
 url="https://github.com/unified-font-object/ufoNormalizer"
-license=(BSD)
-depends=(python)
-makedepends=(python-{build,installer,wheel}
-             python-setuptools-scm)
+license=(BSD-3-Clause)
+depends=(python
+         python-setuptools-scm)
```

Comment:
This bit doesn't seem right. It's possible for setuptools to be a runtime dependency (is it actually in this case) but it is almost impossible for the scm variant to be needed at runtime. If this actually does need setuptools at runtime, lets set setuptools as the dependency and keep setuptools-scm in the build time only dependency list.

Reply:
```log
Checking python-ufonormalizer-0.6.1-4-any.pkg.tar.zst
python-ufonormalizer E: Dependency python-setuptools-scm detected and not included (python modules ['setuptools_scm.get_version'] needed in files ['usr/lib/python3.11/site-packages/ufonormalizer/__init__.py'])
python-ufonormalizer W: Dependency included, but may not be needed ('python-setuptools')
```

Reply:
Did you look at the script itself? Namcap does some simple grep-like searches but doesn't actually follow along with Python logic. This line is in a `try:` statement that is allowed to fail. And actually in this case it should never even *reach* this line of code because the version should be generated by the build. The code is only really useful for running an un-built/un-installed copy of the source.